quantum/process_keycode/process_unicodemap.c

```diff
@@ -49,7 +49,7 @@ bool process_unicode_map(uint16_t keycode, keyrecord_t *record) {
   if ((keycode & QK_UNICODE_MAP) == QK_UNICODE_MAP && record->event.pressed) {
     const uint32_t* map = unicode_map;
     uint16_t index = keycode - QK_UNICODE_MAP;
-    uint32_t code = pgm_read_dword_far(&map[index]);
+    uint32_t code = pgm_read_dword(&map[index]);
```

Comment:
Yes we are using the non-far version elsewhere. Since we don't support 32-bit controllers, I guess that it's ok. But we have to be careful if we do in the future. 

I don't know why the original code used the far version.